MAUI/Scheduler/resource-view.md

```diff
@@ -325,7 +325,7 @@ private ObservableCollection<SchedulerTimeRegion> GetTimeRegion()
 {% endhighlight %}
 {% endtabs %}
 
-![resource-view-special-timeregion-maui-scheduler](images/resource-view/special-regions-resources-in-maui-scheduler.png)
+![Resource view special timeregion in .NET MAUI Scheduler.](images/resource-view/net-maui-scheduler-special-regions-resources.png)
```

Comment:
Is this meaningful? Please check all images.
<img width="635" alt="image" src="https://github.com/user-attachments/assets/085b87c7-90a4-4c71-98de-6b1b15eebe7d">

Reply:
Provide proper image names